lib/ExpensiMark.js

```diff
@@ -214,6 +214,14 @@ export default class ExpensiMark {
                 regex: /<(del)(?:"[^"]*"|'[^']*'|[^'">])*>(.*?)<\/\1>(?![^<]*(<\/pre>|<\/code>))/gi,
                 replacement: '~$2~'
             },
+            {
+                name: 'quote',
+                regex: /\n?<(blockquote|q)(?:"[^"]*"|'[^']*'|[^'">])*>([\s\S]*?)<\/\1>(?![^<]*(<\/pre>|<\/code>))/gi,
```

Comment:
Can you explain what's going on with this portion of the regex?

```
(?:"[^"]*"|'[^']*'|[^'">])
```

Is this just to allow for attributes in the opening HTML tag? If so, can you include some such examples in the unit test?

---

I also have a question about the second capturing group:

```
([\s\S]*?)
```

Is the `?` just so that it doesn't go beyond the next opening tag? If so, do you think this is maybe simpler to understand?

```
([^<]*)
```

Reply:
> (?:"[^"]*"|'[^']*'|[^'">])

Yes, attributes. I will add tests

> ([\s\S]*?)

 here we are matching everything. *? is non-greedy so it will end the match at first </blockquote>.

> ([^<]*)

Can't use it it does not match other HTML tags. Blockquote can have other tags e.g. `b,a del, em` etc.